containers/podman-desktop
# feat: allow to toggle on/off the proxy

### What does this PR do?

Podman Desktop is now storing in its preferences the proxy settings and if it is enabled or not.
Extensions can read this information and then update the proxy configuration

Settings page allows to toggle the proxy configuration

### Screenshot/screencast of this PR
https://user-images.githubusercontent.com/436777/205955418-670bc37c-a74f-40ef-bc60-8d9d013aa0dc.mp4


### What issues does this PR fix or reference?
Fixes #319 


### How to test this PR?

Launch Podman Desktop and goes into Proxy settings
enable the Proxy settings and then check that in `$HOME/.config/containers/containers.conf` you have the proxy settings
toggle the proxy settings and then check that in `$HOME/.config/containers/containers.conf` you don't have the proxy settings 

If you restart it should also keep the settings
